crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -0,0 +1,102 @@
+pub mod common;
+
+mod get_transaction_by_hash_integration_tests {
+    use std::sync::Arc;
+
+    use starknet_core::constants::{DECLARE_V1_TRANSACTION_HASH, DECLARE_V2_TRANSACTION_HASH};
+    use starknet_rs_accounts::{Account, SingleOwnerAccount};
+    use starknet_rs_core::chain_id;
+    use starknet_rs_core::types::contract::{CompiledClass, SierraClass};
+    use starknet_rs_core::types::{
+        BlockId, BlockTag, BroadcastedDeclareTransactionV1, FieldElement,
+    };
+    use starknet_rs_providers::Provider;
+    use starknet_rs_signers::{LocalWallet, SigningKey};
+    use starknet_types::felt::Felt;
+    use starknet_types::traits::ToHexString;
+
+    use crate::common::constants::{
+        CASM_COMPILED_CLASS_HASH, PREDEPLOYED_ACCOUNT_ADDRESS, PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+    };
+    use crate::common::util::BackgroundDevnet;
+
+    #[tokio::test]
+    async fn get_declere_v1_transaction_by_hash_happy_path() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+        let json_string = std::fs::read_to_string(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/test_data/rpc/declare_v1.json"
+        ))
+        .unwrap();
+        let declare_txn_v1: BroadcastedDeclareTransactionV1 =
+            serde_json::from_str(&json_string).unwrap();
+
+        let add_transaction = devnet
```

Comment:
`add_transaction` and `get_transaction` are bad (misleading) names for an object. Those are function names. Object names should have adjectives.

Reply:
true, fixed in f24f6a0e1f320b0210bec8a83cc5132282769419

Reply:
you provided a wrong commit hash/link

Reply:
I see this is the commit: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/75/commits/3b92c3586bb6c1d0702db518f137dcbb7b360494
Resolving this.